File: leetcode/0229-majority-element-ii/solution.cpp
class Solution {
public:
    vector<int> majorityElement(vector<int>& nums) {
        unordered_map<int,int> counter;
        vector<int> ans;
        int n=nums.size();
        for (int i:nums){
            counter[i]++;
        }
        for (auto it:counter){
            if(it.second>(int)n/3){
                ans.push_back(it.first);
            }
        }
        return ans;
    }
};

File: leetcode/0229-majority-element-ii/solution.py
class Solution(object):
    def majorityElement(self, nums):
        n=len(nums)
        n=int(n/3)
        ans=[]
        count={}
        for i in nums:
            if i not in count:
                count[i]=1
                continue
            count[i]+=1
        for key,val in count.items():
            if (val>n):
                ans.append(key)
        return ans
        """
        :type nums: List[int]
        :rtype: List[int]
        """
        
